timeflux_dsp/nodes/peaks.py

```diff
@@ -128,6 +128,11 @@ def update(self):
                                 f'{self.i.data.shape[1]}. Considering the first one. ')
             self.i.data = self.i.data.take([0], axis=1)
 
+        # If there are no no-NaN rows in the column we're considering, there is nothing to do.
+        self.i.data = self.i.data.dropna()
```

Comment:
I'd prefer the user to drop NaNs outside this Node. 
We could add a warning when receiving a chunk with NaNs and skip it if it makes the update failing.